### ISSUE TITLE
Change positioning of the tooltip in advanced settings

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#190]
## User story
1. As a user
2. I want the tooltip positioned accordingly
3. So that I can read the whole text

## Acceptance criteria
* all tooltips in the advanced settings page should be positioned that the user can read the whole text

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
